src/test/java/com/google/javascript/cl2dts/types.js

```diff
@@ -50,3 +50,12 @@ types.h = {a: 'a', b: 34};
  * @template T
  */
 types.identity = function(a) { return a; };
+
+/** @type {Object<number, string>} */
+types.j = {a: 'a'};
+
+/**
+ * @param {Object<K,V>} obj The object in which to look for val.
+ * @template K,V
+ */
+types.containsValue = function(obj) { };
```

Comment:
yeah, I've been in a hurry here trying to get a demo working. agree these names would be good documentation